src/main/java/org/folio/md/validator/service/impl/PermissionDefinitionValidator.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.md.validator.service.impl;
+
+import org.folio.md.validator.model.ValidationContext;
+import org.folio.md.validator.service.Validator;
+
+public class PermissionDefinitionValidator implements Validator {
+
+  @Override
+  public void validate(ValidationContext ctx) {
+    ctx.getModuleDescriptor().getProvides()
```

Comment:
possible NPE

Reply:
moduleDescriptor cannot be null as well [as provides](https://github.com/folio-org/folio-module-descriptor-validator/blob/4b0fae0140b1f33b4ed90089dbc965a58bf3f0af/src/main/java/org/folio/md/validator/model/ModuleDescriptor.java#L22)